helm-charts/splunk-otel-collector/templates/NOTES.txt

```diff
@@ -73,3 +73,6 @@ Splunk Network Explorer is installed and configured.
 [WARNING] "clusterReceiver.eventsEnabled" parameter is deprecated. Soon it will be replaced with "clusterReceiver.k8sObjects".
           Upgrade guidelines: https://github.com/signalfx/splunk-otel-collector-chart/blob/main/UPGRADING.md#$CURRENT_VERSION-to-$NEXT_VERSION
 {{ end }}
+{{- if and (ne .Values.distribution "openshift") (not (empty .Values.securityContextConstraintsOverwrite)) }}
+[WARNING] "securityContextConstraintsOverwrite" paramter should be only used when distribution=openshift.
+{{ end }}
```

Comment:
Not sure it's needed here. If you want to prohibit the usage of `securityContextConstraintsOverwrite` with `distribution != openshift` it's probably better to define the rule in `values.schema.json`